cli.js

```diff
@@ -19,8 +19,12 @@ const cli = meow(`
 	  --padding           Space between the text and box border
 	  --margin            Space around the box
 	  --center            Center the box
-	  --align             Align the text [left|center|right] (Default: left)
-	  --width             Set a fixed width for the box
+	  --align, -a         Align the text [left|center|right] (Default: left)
+	  --title, -t         Display a title at the top of the box.
+	  --title-alignment   Align the title in the top bar [left|center|right].
+	  --width, -w         Set a fixed width for the box
+	  --height, -h        Set a fixed height for the box
+	  --fullscreen, -f    Fit all available space within the terminal.
```

Comment:
I don't want short aliases for these flags. A box is not something you write by hand often, so there's no need for short aliases. It would also be inconsistent to only have it for these and not all.